src/util/OGCFilter.js

```diff
@@ -298,40 +340,49 @@ Ext.define('GeoExt.util.OGCFilter', {
 
             switch (operator) {
             case '==':
-                ogcFilterType = 'PropertyIsEqualTo';
+                ogcFilterType = (isWfs20 ? 'fes:' : '') +
```

Comment:
Introducing a `wfsPrefix` variable and using them in the following lines would make the multiple ternary operators obsolete.

``` var wfsPrefix = (isWfs20 ? 'fes:' : ''); ```

Reply:
Alright, changed in 67d84b9f7c